testing/btest/spicy/spicyc.spicy

```diff
@@ -0,0 +1,11 @@
+# @TEST-DOC: Smoke test for a bundled Spicy.
+#
+# @TEST-REQUIRES: test -x ${BUILD}/auxil/spicy/spicy/bin/spicy-dump
```

Comment:
I'm wondering if we should skip the `requires` to catch of spicy didn't build for some reason. I believe generally the Zeek tests assume having a default built, without specific `configure` options, so Spicy should now always be available.

Reply:
We currently disable Spicy for sanitizer builds (there are three of them now). Spicy doesn't handle sanitizers other than `address` well since currently it assumes that if a sanitizer build is enabled as `address` being active.

Let's keep this conditional for now. Like discussed offline I'll introduce a dedicated script to detect whether Spicy was enabled during configure time.